x/group/internal/orm/types.go

```diff
@@ -113,12 +113,12 @@ func NewTypeSafeRowGetter(prefixKey [2]byte, model reflect.Type, cdc codec.Codec
 		}
 
 		pStore := prefix.NewStore(store, prefixKey[:])
-		it := pStore.Iterator(PrefixRange(rowID))
-		defer it.Close()
-		if !it.Valid() {
+		found := pStore.Has(rowID)
```

Comment:
Is there a reason to perform a `Has` followed by a `Get`? This is gonna cost the user more gas. Why not just do a `Get` and only decode when `bz` is non-empty?

Reply:
good point, I think that should be safe to do it the way you suggest since this is used for tables and not for indexes (in the case of indexes, the values are empty).